### ISSUE TITLE
Add Multiple References Per Region

### ISSUE THREAD

**Issue by alexfinnarn** [alexfinnarn/node_layout#13]
The Layout tool in Express allowed the user to add as many references to other content in regions as they would like.

This was done in a multiple field values way that doesn't exist in Backdrop for entity references. While some reference module should be used eventually in this module, to start, this feature will add an "Add Block" field at the top of the form, and then add each reference to the corresponding region when selected.

**Comment by alexfinnarn** [comment]
While looking at the draggable JS stuff for Backdrop, I got scared. So, I went back to look at what Vue provides component-wise.

https://github.com/SortableJS/Vue.Draggable looks pretty good to me for trying out. https://github.com/David-Desmaisons/draggable-example/blob/master/src/components/Hello.vue is the actual component code.

I've taken some code from the first version of lil_shrugger and added that component as a form element of type `#markup`. The Vue component renders and functions with the ability to pass PHP variables before page load. 

I think using this method, the form security and other options of the Form API are preserved, and the UI is kept in a cleaner state with cleaner code.
